tests/http/test_19_shutdown.py

```diff
@@ -77,13 +76,14 @@ def test_19_02_check_shutdown(self, env: Env, httpd, proto):
             'CURL_DEBUG': 'ssl,tcp,lib-ids,multi'
         })
         curl = CurlClient(env=env, run_env=run_env)
-        url = f'https://{env.authority_for(env.domain1, proto)}/data.json?[0-1]'
+        port = env.port_for(alpn_proto=proto)
+        url = f'https://{env.domain1}:{port}/data.json?[0-1]'
         r = curl.http_download(urls=[url], alpn_proto=proto, with_tcpdump=True, extra_args=[
             '--parallel'
         ])
         r.check_response(http_status=200, count=2)
         assert r.tcpdump
-        assert len(r.tcpdump.stats) == 0, 'Unexpected TCP RSTs packets'
+        assert len(r.tcpdump.get_rsts(ports=[port])) == 0, 'Unexpected TCP RSTs packets'
```

Comment:
`RST` without the "s" maybe, since packets is already plural? On other lines as well.

Reply:
Will fix.